repl/src/main/scala/ammonite/repl/Main.scala

```diff
@@ -70,21 +80,104 @@ case class Main(predef: String = "",
     res
   }
 
+
+  def runScriptLevelCache(path: Path, args: Seq[String] = Vector.empty,
```

Comment:
starting point for running scripts
Tries to load from cache else calls  `runscript`